### DISCUSSION TITLE
Dataset ordering?

### DISCUSSION THREAD

**Discussion by jmgirard** [PittMethods/appliedml#16] [General]
I think we should either order the datasets on `datasets.Rmd` alphabetically by name or in chronological order of their use in the slides. @ShirleyBWang, do you have a preference?

**Comment by ShirleyBWang** [comment]
I like alphabetically!

**Reply by jmgirard** [reply]
Ok, I agree. I reordered the ones in there now, so let's just be careful to maintain alphabetical ordering as we make changes/additions.
